internal/internal_worker.go

```diff
@@ -193,6 +195,15 @@ type (
 		// flags to turn on/off some server side features
 		FeatureFlags FeatureFlags
 	}
+
+	JWTClaims struct {
```

Comment:
Is this the right place to place this struct @longquanzheng ? Or do you have a file where you share common structs between server and client?

Reply:
This is not really going to be shared. I think we just need to make sure the structure is the same as server by contract. I would like to put it under auth/ or new pkg under it if any cycle dependency happens. Because this is an option feature and having things centralized in this package would help us to maintaining the code base. This internal package is already sorta exploding.